docs/operator.rst

```diff
@@ -1338,6 +1338,69 @@ However the ``set`` operator is more idiomatic in Nextflow scripting, since it c
 of a chain of operator transformations, thus resulting in a more fluent and readable operation.
 
 
+subscribe
+---------
```

Comment:
It should go between `splitText` and `sum`

Reply:
Done!